plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts

```diff
@@ -41,6 +42,21 @@ import {
   getPerformanceNavigationEntries,
 } from './utils';
 
+export interface DocumentLoadCustomAttributeFunction {
+  (span: Span): void;
+}
+
+/**
+ * DocumentLoadPlugin Config
+ */
+export interface DocumentLoadInstrumentationConfig
+  extends InstrumentationConfig {
+  /** Functions for adding custom attributes on the spans */
+  applyCustomAttributesOnDocumentLoadSpan?: DocumentLoadCustomAttributeFunction;
+  applyCustomAttributesOnDocumentFetchSpan?: DocumentLoadCustomAttributeFunction;
+  applyCustomAttributesOnResourceFetchSpans?: DocumentLoadCustomAttributeFunction;
+}
```

Comment:
Can you please move these definitions into a `types.ts` file so it is consistent with all other instrumentations?